source/globus/access.rst

```diff
@@ -0,0 +1,24 @@
+
+Log in with your institution:
+
+Visit `www.globus.org <https://www.globus.org>`_ and click :bgrnd1:`Login` at the top of the page. On the Globus login page, choose an organization you are already registered with, such as your school or your employer. Alternatively, one could directly navigate to `globus.org/app/login <https://www.globus.org/app/login>`_.
+
+
+.. image:: img/access-login-screen.png
```

Comment:
Perhaps it is easier to keep track of images when they are in a directory with the same name as the page they are included from?  The advantage is that when you need to move a page, it is immediately clear which images should be moved.
The disadvantage is that sharing images between pages is weird with this approach, but we didn't come across this in the documentation so far.